src/cls/rbd/cls_rbd.cc

```diff
@@ -3097,6 +3097,7 @@ static const std::string IMAGE_KEY_PREFIX("image_");
 static const std::string GLOBAL_KEY_PREFIX("global_");
 static const std::string STATUS_GLOBAL_KEY_PREFIX("status_global_");
 static const std::string INSTANCE_KEY_PREFIX("instance_");
+static const std::string MIRROR_IMAGE_MAP_KEY_PREFIX("map_image_");
```

Comment:
Wouldn't that result in these entries being picked by when `IMAGE_KEY_PREFIX` is used as the key? I had faced this earlier...